cider/tests/utils/CiderTestBase.cpp

```diff
@@ -168,7 +168,8 @@ void CiderJoinTestBase::assertJoinQueryRowEqual(const std::string& sql,
 void CiderJoinTestBase::assertJoinQueryRowEqualForArrowFormat(
     const std::string& sql,
     const std::string& json_file,
-    const bool ignore_order) {
+    const bool ignore_order,
+    const bool compare_value) {
   VLOG(4) << sql;
```

Comment:
remove this?

Reply:
This is debug4 level log to help identify the sql when using ```define``` for test. And in cider codegen part, it also contains many debug level log.